app/src/main/kotlin/com/simplemobiletools/calendar/pro/helpers/Constants.kt

```diff
@@ -159,7 +159,8 @@ const val LAST_MODIFIED = "LAST-MODIFIED"
 const val DTSTAMP = "DTSTAMP:"
 const val DURATION = "DURATION:"
 const val SUMMARY = "SUMMARY"
-const val DESCRIPTION = "DESCRIPTION:"
+const val DESCRIPTION = "DESCRIPTION"
```

Comment:
This change has broken the export.

Reply:
acknowledged. 
currently working on it.

Reply:
strike that.   export doesn't work with current master branch in the first place.  i just did a fresh install and build from current master branch.   export is already broken.  id like to fix it, however it is important to note my patch does not break it.

Reply:
I checked on the master branch and export is working. I mean exporting "normal text" descriptions, not HTML ones. In ICS files after your change `:` is missing after `DESCRIPTION`.

Reply:
okay, now i am really confused.  master branch, current.  when i got to hamburger (the three dots) select "Export Events to an .ics file"     i get an error, "no entries for exporting"  if i import an ics file from hamburger, it doesnt work.    if i import from a file manger it imports.

what are you doing different??  hold my hand for a moment.

Reply:
I have a fresh copy of the app running on an emulator (latest master). I've created one event with description, then export from the three dots menu. On your branch, export is also working, but the `DESCRIPTION` in .ics is malformed.

Reply:
(i am using an actual phone by the way. android 13, galaxy a71)
"On your branch, export is also working, but the DESCRIPTION in .ics is malformed."   ohh...   i get an exported file with 0 bytes.  when using master current.  my patch, i get the same result.      i wish i could get a malformed ics.  id have something to work from.

i am currently downloading the android 13 emulator/virtual device, "tiramisu" under android studio.  ( what is the standard to use???)  what is your setup for virtual/emulator 

i'd really like to see what is going on here.    it seems both our testing come back different, slightly.

in the meantime, do you want to try something?
in `helpers/IcsExporter.kt`
line 90: `writeLn("$DESCRIPTION_EXPORT$reminderLabel")`
line 121:  `out.writeLn("$DESCRIPTION_EXPORT$substring")`

in `helpers/Constants.kt`
add
`const val DESCRIPTION_EXPORT = "DESCRIPTION:"`

Reply:
could you please????   send me an example ics with the exported ics ?? from master branch and one from my patch??

Reply:
on the emulator, this works. and DESCRIPTION is correct in the file. from MY tests.
in `helpers/IcsExporter.kt`
line 90: `writeLn("$DESCRIPTION_EXPORT$reminderLabel")`
line 121: `out.writeLn("$DESCRIPTION_EXPORT$substring")`

in `helpers/Constants.kt`
add
`const val DESCRIPTION_EXPORT = "DESCRIPTION:"`

on the actual phone.   no dice.  i get an export file with 0 bytes.
emulator works great, i think i'll just use the emulator instead.    

i don't know what to make of this now.  any ideas, on why my phone and probably many other will not work correctly?  or is my phone the problem??  not sure what to work on.  let me know your findings, please?

Reply:
I've tested now both on emulator and real phone (OnePlus 8 Pro with Android 13) - export+import, in the way I've described, works. ICS file is populated, so I don't know, maybe that's some issue related to your phone - e.g. insufficient permissions? Maybe Samsung's OS somehow limits debug versions of the apps? I really don't know.
The bug I've raised is now fixed, so from my perspective, it's good. Now your PR should be confirmed by @tibbi if it's OK for him and if it solves the bug it was originally made to fix. Good luck 🙂

Reply:
yeah it must be my phone.  i have it locked down pretty tight.   well, thank you for your help!

Reply:
> I've tested now both on emulator and real phone (OnePlus 8 Pro with Android 13) - export+import, in the way I've described, works. ICS file is populated, so I don't know, maybe that's some issue related to your phone - e.g. insufficient permissions? Maybe Samsung's OS somehow limits debug versions of the apps? I really don't know. The bug I've raised is now fixed, so from my perspective, it's good. Now your PR should be confirmed by @tibbi if it's OK for him and if it solves the bug it was originally made to fix. Good luck slightly_smiling_face

so... this is inaccurate?